### ISSUE TITLE
StreamWrapper unknown method stream_seek, stream_tell...

### ISSUE THREAD

**Issue by AstRonin** [yiisoft/yii2-mongodb#147]
### What steps will reproduce the problem?

Need use `Response::sendContent()` and set `Response::stream` by array(). We should call `fseek` for this test. 
For example, we can save video file to gridFS and open video in browser.

### What's expected?

loading file in browser

### What do you get instead?

Error: yii\mongodb\file\StreamWrapper unknown method stream_seek
Actually, method `stream_tell` unknown too, and maybe some other...

### Additional info

| Q                      | A
| ---------------------- | ---
| Yii version            | 2.0.9
| Yii MongoDB version    | 2.1.0
| MongoDB server version | 3.2
| PHP version            | 5.6
| Operating system       | Win64

And I have a question.
Why all properties in `yii\mongodb\file\StreamWrapper` are private? Instead of extend this class I was forced create own class.

I added next implementation of non-existent methods, maybe not full, please check if needed.

```php
public function stream_seek($offset, $whence = SEEK_SET) {
        switch ($whence) {
            case SEEK_SET:
                if ($offset < $this->download->getSize() && $offset >= 0) {
                     $this->pointerOffset = $offset;
                     return true;
                } else {
                     return false;
                }
                break;

            case SEEK_CUR:
                if ($offset >= 0) {
                     $this->pointerOffset += $offset;
                     return true;
                } else {
                     return false;
                }
                break;

            case SEEK_END:
                if ($this->download->getSize() + $offset >= 0) {
                     $this->pointerOffset = $this->download->getSize() + $offset;
                     return true;
                } else {
                     return false;
                }
                break;

            default:
                return false;
        }
    }
    
    public function stream_tell() {
        return $this->pointerOffset;
    }
```

**Comment by klimov-paul** [comment]
Can you open a PR for adding missing methods?

**Comment by AstRonin** [comment]
Yes, opened. I hope it's normal PR

**Comment by AstRonin** [comment]
Ok :) Another one.

**Comment by klimov-paul** [comment]
Resolved by commit 9e4a363
